app/models/item.rb

```diff
@@ -1,4 +1,6 @@
 class Item < ApplicationRecord
   belongs_to :wishlist
   validates :name, presence: true
+
+  scope :change_reserved, ->(item, is_reserved) { item.update_attributes(is_reserved: is_reserved) }
```

Comment:
That's not what a scope is for. [Read up on `scope`](http://guides.rubyonrails.org/active_record_querying.html#scopes) and change this to a normal method which should prolly be named `update_reserved` to keep with the Rails convention.